frontend/packages/erd-core/src/components/ERDRenderer/convertDBStructureToNodes.ts

```diff
@@ -43,7 +43,7 @@ export const convertDBStructureToNodes = ({
         targetColumnCardinalities: tableColumnCardinalities.get(table.name),
       },
       position: { x: 0, y: 0 },
-      zIndex: 1,
+      zIndex: 2,
```

Comment:
💡 
Personally, I'm not very familiar with such frontend issues, but defining a file like `src/components/ERDRenderer/z-index.ts` or `src/components/z-index.ts` for cross-component definitions might prevent future omissions or oversights. It seems like a good idea. This approach could be suitable if you frequently call from `.ts` or `.tsx` files. 
However, it might be difficult to use from a `.css` file. What do you think? @liam-hq/liam-dev 

e.g.

```typescript
export const zIndex = {
  tableNode: 2,
  highlightEdge: 1,
  unhighlightEdge: 0,
}
```

```typescript
// when using
import { zIndex } from '../z-index'

// ...

zIndex: zIndex.tableNode
```

Reply:
It would be better to define the z-index overlap as a constant 👍🏻 
There is a problem that it cannot be referenced in CSS, but as long as it is used as a property of React Flow, it seems to be fine to define it in TypeScript.

Reply:
That's true!
zIndex was a magic number and not good.

Is this general to define constants? see below:
1f10080cb927c839851717f0600f75d6b8e94ac7